engine/src/main/java/org/terasology/engine/rendering/opengl/FBO.java

```diff
@@ -728,6 +728,9 @@ public int height() {
          * @param height An integer representing the new height.
          */
         public void setDimensions(int width, int height) {
```

Comment:
that's just obscuring the problem by having the method silently fail.

Reply:
the precondition should hit and give you a backtrace and give you information where this is coming from. I don't know off the top of my head why its getting set to 0,0